src/plugins/remove/azureRemovePlugin.ts

```diff
@@ -40,10 +41,25 @@ export class AzureRemovePlugin extends AzureBasePlugin {
   }
 
   private async remove() {
-    const resourceClient = new ResourceService(this.serverless, this.options);
-    await resourceClient.deleteDeployment();
-    await resourceClient.deleteResourceGroup();
-
-    this.log("Service successfully removed");
+    const resourceService = new ResourceService(this.serverless, this.options);
+    const rg = await resourceService.getResourceGroup();
+    const rgName = resourceService.getResourceGroupName();
+    if (!rg) {
+      this.log(`Resource group "${rgName}" does not exist in your Azure subscription`)
+      return;
+    }
+    this.log(`This command will delete your entire resource group (${resourceService.getResourceGroupName()}). ` +
+      "Are you sure you want to proceed? If so, enter the full name of the resource group :");
+    const input = await Utils.waitForUserInput();
+    if (input === resourceService.getResourceGroupName()) {
+      this.log("Deleting resource group");
+      await resourceService.deleteDeployment();
```

Comment:
What does `deleteDeployment()` do?  Do we even need it if we are removing the whole resource group?

Reply:
I was actually wondering the same thing... I don't remember implementing this (although it's possible that I still did), but I didn't want to change the functionality in this PR without finding out why it was there